sentry-types/src/protocol/profile.rs

```diff
@@ -0,0 +1,110 @@
+use super::v7::{DebugMeta, TraceId};
+use serde::{Deserialize, Serialize, Serializer};
+use uuid::Uuid;
+
+fn serialize_id<S: Serializer>(uuid: &Uuid, serializer: S) -> Result<S::Ok, S::Error> {
+    serializer.serialize_some(&uuid.as_simple().to_string())
```

Comment:
```suggestion
    serializer.serialize_some(&uuid.as_simple())
```

`Simple` implements `Serialize`: https://docs.rs/uuid/latest/uuid/fmt/struct.Simple.html#impl-Serialize